.dev_scripts/benchmark_train.py

```diff
@@ -13,15 +13,69 @@
 from mmdet.utils import register_all_modules, replace_cfg_vals
 
 
+# TODO: Support full ceph
+def replace_to_ceph(cfg):
```

Comment:
add a file to put this function, and import it in test and train scripts

Reply:
OK